ui-manchette/src/components/OperationalPointList.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react';
+import { OperationalPointType } from '../types/pathPropertiesTypes';
+import OperationalPoint from './OperationalPoint';
+import cx from 'classnames';
+import { positionKm } from './helpers';
+
+interface OperationalPointListProps {
+  operationalPoints: OperationalPointType[];
+  isProportionnal?: boolean;
+  zoom?: number;
+  width?: number;
+}
+
+const OperationalPointList: React.FC<OperationalPointListProps> = ({
+  operationalPoints,
+  isProportionnal = false,
+  zoom = 1,
+}) => {
+  const [operationalPointsToDisplay, setOperationalPointsToDisplay] = React.useState<
+    OperationalPointType[]
+  >([]);
+  const baseHeight = 32;
```

Comment:
Move this const outside the component, and maybe use screaming case
```suggestion
  const BASE_HEIGHT = 32;
```